web/src/features/fba/components/infoPanel/FireZoneUnitTabs.tsx

```diff
@@ -0,0 +1,170 @@
+import { Box, Grid, Tab, Tabs, Tooltip } from '@mui/material'
+import { FireCenter, FireCentreHfiFuelsData, FireShape, FireShapeAreaDetail, FireZoneTPIStats } from 'api/fbaAPI'
+import { INFO_PANEL_CONTENT_BACKGROUND, theme } from 'app/theme'
+import FireZoneUnitSummary from 'features/fba/components/infoPanel/FireZoneUnitSummary'
+import InfoAccordion from 'features/fba/components/infoPanel/InfoAccordion'
+import TabPanel from 'features/fba/components/infoPanel/TabPanel'
+import { ADVISORY_ORANGE_FILL, ADVISORY_RED_FILL } from 'features/fba/components/map/featureStylers'
+import { useFireCentreDetails } from 'features/fba/hooks/useFireCentreDetails'
+import { isNull, isUndefined } from 'lodash'
+import React, { useEffect, useMemo, useState } from 'react'
+
+interface FireZoneUnitTabs {
+  selectedFireZoneUnit: FireShape | undefined
+  setZoomSource: React.Dispatch<React.SetStateAction<'fireCenter' | 'fireShape' | undefined>>
+  fireCentreTPIStats: Record<string, FireZoneTPIStats[]> | null
+  fireCentreHfiFuelTypes: FireCentreHfiFuelsData
+  selectedFireCenter: FireCenter | undefined
+  advisoryThreshold: number
+  setSelectedFireShape: React.Dispatch<React.SetStateAction<FireShape | undefined>>
+}
+
+const calculateStatus = (details: FireShapeAreaDetail[], advisoryThreshold: number) => {
```

Comment:
It looks like we have three similar versions of this `calculateStatus` function. It would be nice to consolidate if possible.

Reply:
Done in 78badf1

Reply:
Nice that brought up code coverage by a percentage point 😎